src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -152,6 +150,9 @@ def collect_data(geop_result, geojson):
 
     z['SedNitr'] = geop_result['soiln'] * 4.0
 
+    recess_coef = geop_result['recess_coef'] * -0.0015 + 0.1103
```

Comment:
Consider moving these calculations to within the `recess_coef` and `soilp` (and `soiln`) methods. Unless a value is assigned multiple times, we tend to not do any calculations here, instead doing everything within the "process" method after the geoprocessing. This helps keep these magic numbers close to original references, and they can be expounded on via comments without clutter.